### ISSUE TITLE
Clean up old Rubies and Rails

### ISSUE THREAD

**Issue by maurogeorge** [michaeldv/awesome_print#189]
> clean up the code to get rid of Ruby 1.8.x-only hacks. I think by now it's safe to drop support for 1.9.3 as well and focus on Ruby v2.x only. I think we can certainly drop Rails v2 but I'm not so sure about dropping Rails v3.

Created based on the idea of @michaeldv [about the future of the project](https://github.com/michaeldv/awesome_print/pull/161#issuecomment-67369830)

**Comment by imajes** [comment]
Yeah i think that's a good idea. makes sense to signpost on README that the current release is the last one that supports 1.8x and Rails 2x.

**Comment by gerrywastaken** [comment]
Sounds good to me.

@waldyr How do you feel about us dropping support for anything below Ruby 2.0?

We are already running into build issues on Travis every time somebody drops support in their gem resulting in the addition of many `gem <old version goes here>, :platforms => :ruby_19` lines to our Appraisals file.
https://github.com/awesome-print/awesome_print/blob/master/Appraisals

Just yesterday the json gem's dropped support for old rubies resulted in failed builds: https://github.com/awesome-print/awesome_print/pull/255

I suggest officially dropping support for 1.9 after the next release and removing/updating redundant code as convenient in a piecemeal fashion.

**Comment by waldyr** [comment]
I agree with that. 👍

**Comment by gerrywastaken** [comment]
@waldyr Cheers

**Comment by waldyr** [comment]
Also this is one thing @michaeldv pointed out it should be done :)
